### ISSUE TITLE
Improve fingerprint usability when adding an account

### ISSUE THREAD

**Issue by arlolra** [arlolra/ctypes-otr#76]
Since #75, when adding *an account* (rather than a contact), I'm asked if I have a fingerprint for each contact associated with that account.  For an account with lots of contacts, it can get tedious skipping them all.

We should probably avoid triggering the manual fingerprint window when adding an account.

/cc @c633 

There's also cleaning up the context menu still TODO
https://github.com/arlolra/ctypes-otr/pull/75#issuecomment-220377663

**Comment by c633** [comment]
How does it trigger the fingerprint window when adding an account? I tried with my twitter account but after adding my account, nothing showed up.

**Comment by arlolra** [comment]
Yeah, the Twitter implementation is kind of odd in that it is modelled as a multi-user chat, similar to an IRC chatroom, with you and those you're following.  Following someone there doesn't equate to adding them as a contact, but more of a join to the room.

This can best be tested with an XMPP account, that already has some contacts stored in the roster on the server.

**Comment by clokep** [comment]
"doesn't equate to adding them as a contact" is a bug that we've wanted fixed for a long time. It should be trivial to implement, but doesn't make much sense until direct messages are supported.

**Comment by arlolra** [comment]
@clokep Thanks for the reminder.  I need to get on rebasing https://bugzilla.mozilla.org/show_bug.cgi?id=955642

**Comment by c633** [comment]
> We should probably avoid triggering the manual fingerprint window when adding an account.

Agree. I will create a PR for this issue in the next few days.

**Comment by c633** [comment]
Did you have any solutions for this issue? @arlolra

**Comment by arlolra** [comment]
I haven't looked into it yet, but thanks for reminding me.  We should get this fixed before the next release.
